networking/hardware_networks/using-pod-level-bonding.adoc

```diff
@@ -0,0 +1,23 @@
+[id="using-pod-level-bonding"]
+= Using pod-level bonding
+include::modules/common-attributes.adoc[]
+:context: using-pod-level-bonding
+
+toc::[]
+
+
+:FeatureName: Bond Container Network Interface (CNI)
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Bonding at the pod level is vital to enable workloads inside pods that require high availability and more throughput. With pod-level bonding, you can create a bond interface from multiple single root I/O virtualization (SR-IOV) virtual function interfaces in kernel mode interface). The SR-IOV virtual functions are passed into the pod and attached to a kernel driver.
```

Comment:
> virtual function interfaces in kernel mode interface).
There's a dangling parentheses, I can't see where it should start. Please correct or delete.